<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PsdDialog</class>
 <widget class="QDialog" name="PsdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1720</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color:rgb(217, 217, 217)}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="900,700">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{background-color:rgb(170, 170, 170)}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_13">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>300</width>
        <height>300</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame
{background-color:rgb(0, 0, 0);}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>18</y>
         <width>264</width>
         <height>264</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{background-color:rgb(0, 170, 0)}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>2</y>
          <width>260</width>
          <height>260</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{background-color:rgb(0, 0, 0)}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_29">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>131</y>
          <width>264</width>
          <height>2</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{background-color:rgb(0, 170, 0)}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame_30">
        <property name="geometry">
         <rect>
          <x>131</x>
          <y>0</y>
          <width>2</width>
          <height>264</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{background-color:rgb(0, 170, 0)}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>300</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>Horizontal</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>300</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>8.000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>210</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>Deflection:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>140</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>230</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>140</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>4.000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>Signal Sum:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>Deflection:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>230</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>4.000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>Vertical</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>404</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{background-color:rgb(103, 103, 103)}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_10">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>2</y>
         <width>400</width>
         <height>20</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{background-color:rgb(0, 161, 0)}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>400</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>8.000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>400</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_11">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>100</y>
        <width>64</width>
        <height>304</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{background-color:rgb(103, 103, 103)}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_14">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>2</y>
         <width>60</width>
         <height>300</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{background-color:rgb(0, 161, 0)}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <zorder>frame_11</zorder>
      </widget>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>70</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Retraced</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>250</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>8.000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>220</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Z Piezo Voltage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>250</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
       </font>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_37">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>410</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Extended</string>
      </property>
     </widget>
     <zorder>frame_13</zorder>
     <zorder>label_31</zorder>
     <zorder>label_24</zorder>
     <zorder>label_28</zorder>
     <zorder>label_25</zorder>
     <zorder>label_29</zorder>
     <zorder>label_30</zorder>
     <zorder>label_26</zorder>
     <zorder>label_23</zorder>
     <zorder>label_12</zorder>
     <zorder>label_27</zorder>
     <zorder>label_10</zorder>
     <zorder>frame_9</zorder>
     <zorder>label_32</zorder>
     <zorder>label_33</zorder>
     <zorder>frame_11</zorder>
     <zorder>label_13</zorder>
     <zorder>label_34</zorder>
     <zorder>label_35</zorder>
     <zorder>label_36</zorder>
     <zorder>label_37</zorder>
     <zorder>frame_14</zorder>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{background-color:rgb(170, 170, 170)}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <zorder>frame</zorder>
     <zorder>frame</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
